README.md

```diff
@@ -12,7 +12,42 @@ The aim of govuk-docker is to make it easy to develop any GOV.UK app. It achieve
 
 ## Installation
 
-[Instructions for how to install and setup govuk-docker are here.](docs/installation.md)
+**First make sure you have the following dependencies.**
+
+- [brew](https://brew.sh/)
+- [git](https://git-scm.com)
+- A directory `~/govuk` in your home directory
```

Comment:
Minor: the tilde already implies this is in your home directory. If we're wanting to make it clearer for those who don't understand tilde we could just switch it to `govuk`.

Reply:
Good point. I've reworded this.